docs/api/transforms.md

```diff
@@ -1,4 +1,4 @@
-# Transforms
+# Transforms API
```

Comment:
I don't think we call the other API sections as API, so I would either change all of them or revert this.

Reply:
I've added "API" to all of the API documents, to distinguish them from the Concepts documents.